nixos/tests/acme.nix

```diff
@@ -578,7 +578,7 @@ in {
               webserver.wait_for_unit(f"acme-finished-{test_domain}.target")
               wait_for_server()
               check_connection(client, test_domain)
-              rc, _ = client.execute(
+              rc, _s = client.execute(
```

Comment:
why this change? i do not understand what it does as the variable isn't used

Reply:
It is defined twice though. `_` doesn't seem to get special treatment.

Reply:
oh, sad.